src/Widgets/BridgeWidgetApi.ts

```diff
@@ -180,4 +168,13 @@ export class BridgeWidgetApi {
         await this.connMan.purgeConnection(roomId, connectionId);
         res.send({ok: true});
     }
+
+    private async getConnectionTargets(req: ProvisioningRequest, res: Response) {
+        if (!req.userId) {
+            throw Error('Cannot get connections without a valid userId');
+        }
+        const type = req.params.type as string;
```

Comment:
If that's not guaranteed to be a `string` you shouldn't cast it to a string.

Do you expect `this.connMan.getConnectionTargets` to deal with this? Then please change its signature.

Reply:
I have no idea why I decided to cast this...it is a string.